package.json

```diff
@@ -87,6 +87,16 @@
           }
         ],
         "priority": "default"
+      },
+      {
+        "viewType": "one.editor.jsonTracer",
+        "displayName": "chromeTrace",
```

Comment:
```suggestion
        "displayName": "Chrome Tracer",
```

How about changing the displayName is above?

It's a name that will be used by vscode UI commands, for example,
![Screenshot from 2022-08-05 13-44-40](https://user-images.githubusercontent.com/17171963/183041626-bcd3c30b-0717-473f-8adc-9e36ead97637.png)

Reply:
Make sense :D
But as this is "Json Tracer", not "Chrome", I will change it to `"Json Tracer" :)

Reply:
I will follow the result of #1183 :D
But I agree with capital letters!